Captcha not showing up, missing libfreetype

Hi, 
I set up seafile using your docker compose setup and I noticed that after I typed wrong password oft enough I will get a captcha prompt by no captcha image.

I get this error in my seahub log:
```
2021-06-05 18:58:12,026 [ERROR] django.request:228 log_response Internal Server Error: /captcha/image/d1eb0a433a290465afb68dfcb2ce3fe595a36774/
Traceback (most recent call last):
  File "/opt/seafile/seafile-server-8.0.4/seahub/thirdpart/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/opt/seafile/seafile-server-8.0.4/seahub/thirdpart/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/opt/seafile/seafile-server-8.0.4/seahub/thirdpart/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/opt/seafile/seafile-server-8.0.4/seahub/thirdpart/captcha/views.py", line 67, in captcha_image
    font = ImageFont.truetype(fontpath, settings.CAPTCHA_FONT_SIZE * scale)
  File "/opt/seafile/seafile-server-8.0.4/seahub/thirdpart/PIL/ImageFont.py", line 853, in truetype
    return freetype(font)
  File "/opt/seafile/seafile-server-8.0.4/seahub/thirdpart/PIL/ImageFont.py", line 850, in freetype
    return FreeTypeFont(font, size, index, encoding, layout_engine)
  File "/opt/seafile/seafile-server-8.0.4/seahub/thirdpart/PIL/ImageFont.py", line 185, in __init__
    if core.HAVE_RAQM:
  File "/opt/seafile/seafile-server-8.0.4/seahub/thirdpart/PIL/ImageFont.py", line 44, in __getattr__
    raise ImportError("The _imagingft C module is not installed")
ImportError: The _imagingft C module is not installed
```

It looks like it just needs the libfreetype6 package. Here is an [issue](https://github.com/hanwckf/seafile-arm64-docker/issues/2) and the fixing [commit](https://github.com/hanwckf/seafile-arm64-docker/commit/d767a25d34729fcbc438946b26b515abe53ffc42#diff-c113a16330cef7381de5231f60896f5f96494e3dbaaeba69f54f7812f4f5dbc9) in another seafile docker repository.

--- a/Dockerfile
+++ b/Dockerfile
@@ -51,7 +51,8 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
     libjpeg62-turbo \
     libopenjp2-7 \
     libtiff5 \
-    libxcb1 && \
+    libxcb1 
+    libfreetype6 && \
     rm -rf /var/lib/apt/lists/*
 
 WORKDIR /opt/seafile

--- a/Dockerfile
+++ b/Dockerfile
@@ -51,7 +51,7 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
     libjpeg62-turbo \
     libopenjp2-7 \
     libtiff5 \
-    libxcb1 
+    libxcb1 \
     libfreetype6 && \
     rm -rf /var/lib/apt/lists/*
 

